Ginger/GingerCoreNET/Run/RunSetConfig.cs

```diff
@@ -177,7 +177,6 @@ public void UpdateRunnersBusinessFlowRunsList()
 
         public override void UpdateBeforeSave()
         {
-            UpdateRunnersBusinessFlowRunsList();
```

Comment:
why you removed the UpdateRunnersBusinessFlowRunsList() from UpdateBeforeSave()? with this change we will do the  UpdateRunnersBusinessFlowRunsList()  only in case runset is been saved from the specific save funtion in RunSetPage and not in any case as it should.
please explain why it was needed.

Reply:
UpdateRunnersBusinessFlowRunsList() making RunnersBusinessFlowRunsList empty when we duplicating/ coping runset.
So UpdateRunnersBusinessFlowRunsList() is called from SaveRunSetConfig() to update the RunnersBusinessFlowRunList.